docs/advanced-features/compiler.md

```diff
@@ -9,6 +9,7 @@ description: Learn about the Next.js Compiler, written in Rust, which transforms
 
 | Version   | Changes                                                                                                                            |
 | --------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `v13.1.0` | Modularize Imports [stable](https://nextjs.org/blog/next-13-1#import-resolution-for-smaller-bundles).                              |
```

Comment:
Not sure if this change is necessary. I added it because there’s `SWC Minifier stable` 2 lines below, so I did something similar. But feel free to remove it if this isn’t necessary.